app/uk/gov/ons/sbr/models/VatRef.scala

```diff
@@ -0,0 +1,11 @@
+package uk.gov.ons.sbr.models
+
+case class VatRef(value: String)
+
+object VatRef {
+  def toIdTypePair(vatref: VatRef): (UnitId, UnitType) =
+    asUnitId(vatref) -> UnitType.ValueAddedTax
+
+  def asUnitId(vatRef: VatRef): UnitId =
```

Comment:
I assume this will move to another object in the future as all UnitType will typically have a need for a function like this

Reply:
Not really - as the output is based on the input reference, and we currently have no commonality across references.  So there will be similar code across the reference objects that can convert themselves to the more generic id.  See ERN for example.

Do you have an alternative suggestion?